modules/service-cis.sh

```diff
@@ -0,0 +1,71 @@
+# shellcheck disable=SC2034,SC2039
+ 
+CISAUDIT_SERVICE_TITLE="Canonical CIS Benchmark 16.04 Audit Tool"
+CISAUDIT_SUPPORTED_SERIES="xenial"
+CISAUDIT_SUPPORTED_ARCHS="x86_64 ppc64le s390x"
+
+CISAUDIT_REPO_URL="https://private-ppa.launchpad.net/ubuntu-advantage/security-benchmarks"
+CISAUDIT_REPO_KEY_FILE="ubuntu-securitybenchmarks-keyring.gpg"
+CISAUDIT_REPO_LIST=${CISAUDIT_REPO_LIST:-"/etc/apt/sources.list.d/ubuntu-cis-${SERIES}.list"}
+CISAUDIT_UBUNTU_CISBENCHMARK="ubuntu-cisbenchmark-16.04"
+
+cisaudit_enable() {
+    local token="$1"
+    local result=0
+
+    _cisaudit_is_installed || result=$?
+    if [ $result -eq 0 ]; then
+        error_msg "CIS benchmark audit package is already installed and files are available in /usr/share/ubuntu-securityguides/$CISAUDIT_UBUNTU_CISBENCHMARK."
```

Comment:
When I used CISAUDIT_REPO_URL="https://private-ppa.launchpad.net/fips-cc-stig/fipsdevppa", and ran the ubuntu-advantage tool with the creds, it installed the ubuntu-cisbenchmark-16.04 package with files in /usr/share/ubuntu-securityguides/ubuntu-cisbenchmark-16.04 directory.

Reply:
Indeed it is correct. In fact, I now think this code is never reached, because before _service_check_enabled() is called before cisaudit_enable(). I understand it was copied from another module and can be left as is.